testfiles/2ZoneDataCenterHVAC_wEconomizer.idf

```diff
@@ -3096,7 +3096,11 @@
 
   EnergyManagementSystem:Program,
     CalculatePUE,            !- Name
-    set PUE_Value = Whole_Building_Power / IT_Equip_Power;  !- Program Line 1
+    IF IT_Equip_power > 0.0, !- Program Line 1
+    set PUE_Value = whole_building_power / IT_Equip_power,  !- Program Line 2
+    ELSE,                    !- Program Line 3
+    set PUE_Value = 0.0,     !- Program Line 4
+    ENDIF;                   !- Program Line 5
```

Comment:
So this is the reason for a reverse DD issue? PUE_Value was retained for the next environment?

Reply:
No, the ITE supply air inlet node temp was retained into the next environment, because IT equipment gets simulated in the heat balance, before HVACManager is called, which is where ResetNodeData used to be.

Reply:
Then why was this changed?

Reply:
'cause it blew up with a zero for IT_Equip_Power after the other fixes went in.